src/components/Typography/Text.tsx

```diff
@@ -4,56 +4,47 @@ import {
   ColorProps,
   colorStyle,
   ColorStyleProps,
-  fontFamily,
-  FontFamilyProps,
-  fontSize,
-  FontSizeProps,
   fontStyle,
   FontStyleProps,
-  fontWeight,
-  FontWeightProps,
-  letterSpacing,
-  LetterSpacingProps,
   lineHeight,
   LineHeightProps,
   size,
   SizeProps,
   space,
   SpaceProps,
   textStyle,
-  TextStyleProps
+  TextStyleProps,
+  typography,
+  TypographyProps
 } from 'styled-system';
 
 import { textVariants, TextVariants } from '@theme';
 
 export type TextProps = SpaceProps &
   LineHeightProps &
-  FontSizeProps &
   FontStyleProps &
   SizeProps &
   ColorProps &
   ColorStyleProps &
   TextStyleProps &
-  FontFamilyProps &
-  FontWeightProps &
-  LetterSpacingProps & {
+  TypographyProps & {
     variant?: TextVariants;
     as?: keyof JSX.IntrinsicElements | React.ComponentType<any>;
+  } & {
+    capitalize?: boolean;
   };
 
 const Text: React.FC<TextProps> = styled.p<TextProps>`
+  ${textVariants}
   ${space}
-  ${fontSize}
   ${fontStyle}
   ${color}
   ${size}
   ${colorStyle}
   ${textStyle}
   ${lineHeight}
-  ${letterSpacing}
-  ${fontFamily}
-  ${fontWeight}
-  ${textVariants}
+  ${typography}
```

Comment:
Can you explain this change?

Reply:
From https://styled-system.com/api#typography
`typography` includes `fontFamily, fontSize, fontWeight, lineHeight, letterSpacing, textAlign, and fontStyle.` so I can simplify the amount of rules included.